pkgs/top-level/all-packages.nix

```diff
@@ -22594,14 +22594,14 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
   };
 
-  mongodb-4_4 = callPackage ../servers/nosql/mongodb/4.4.nix {
+  mongodb-4_4 = callPackage ../servers/nosql/mongodb/v4_4.nix {
     sasl = cyrus_sasl;
     boost = boost17x.override { enableShared = false; };
     inherit (darwin) cctools;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
   };
 
-  mongodb-5_0 = callPackage ../servers/nosql/mongodb/5.0.nix {
+  mongodb-5_0 = callPackage ../servers/nosql/mongodb/v5_0.nix {
     sasl = cyrus_sasl;
```

Comment:
Do we even need this? It seems always to be cyrus_sasl.

Reply:
Would it be clear which sasl library should be used if we removed it? There seem to be at least 2 options, namely cyrus_sasl and gsasl.

Reply:
Does mongodb even work with gsasl? If so this should rather be a boolean input to enable one or the other.

Reply:
I think it would be possible to use gsasl:
https://www.mongodb.com/docs/manual/reference/program/mongos/#std-option-mongos.--ldapBindSaslMechanisms